src/Reflection/ParameterInspector.php

```diff
@@ -84,8 +84,8 @@ private function getMethodSignature($className, $methodName, \ReflectionClass $r
                 $parameterSignature = [
                     "name" => $name
                 ];
-                if ($parameter->getClass()) {
-                    $parameterSignature['type'] = $parameter->getClass()->getName();
+                if ($parameter->getType() && !$parameter->getType()->isBuiltin()) {
```

Comment:
`getClass()` is deprecated 
more info https://php.watch/versions/8.0/deprecated-reflectionparameter-methods